src/components/menus/ActivityStatusMenu.svelte

```diff
@@ -0,0 +1,93 @@
+<svelte:options immutable={true} />
+
+<script lang="ts">
+  import ActivitiesIcon from '@nasa-jpl/stellar/icons/activities.svg?component';
+  import { createEventDispatcher } from 'svelte';
+  import type { ActivityDirectiveValidationStatus } from '../../types/activity';
+  import type { ActivityErrorCounts } from '../../types/errors';
+  import { Status } from '../../utilities/status';
+  import PlanNavButton from '../plan/PlanNavButton.svelte';
+  import ActivityErrorsRollup from '../ui/ActivityErrorsRollup.svelte';
+
+  export let activityErrorCounts: ActivityErrorCounts;
+  export let activityDirectiveValidationStatuses: ActivityDirectiveValidationStatus[] = [];
+  export let compactNavMode: boolean = false;
+
+  const dispatch = createEventDispatcher();
+
+  let invalidActivityCount: number = 0;
+  let totalActivitiesCheckedCount: number = 0;
+  let totalActivityCount: number = 0;
+
+  $: {
+    ({ invalidActivityCount, totalActivitiesCheckedCount, totalActivityCount } =
+      activityDirectiveValidationStatuses.reduce(
+        (prevCounts, validationStatus) => {
+          return {
+            invalidActivityCount:
+              prevCounts.invalidActivityCount + (validationStatus.validations.success === false ? 1 : 0),
+            totalActivitiesCheckedCount:
+              prevCounts.totalActivitiesCheckedCount + (validationStatus.status === 'complete' ? 1 : 0),
+            totalActivityCount: prevCounts.totalActivityCount + 1,
+          };
+        },
+        {
+          invalidActivityCount: 0,
+          totalActivitiesCheckedCount: 0,
```

Comment:
Is the "not checked" bit from the mockups something that didn't pan out or didn't make sense in the final the backend/frontend implementation? 
![image](https://github.com/NASA-AMMOS/aerie-ui/assets/4419607/780b69a9-add3-4fd2-bb27-168b9e9e5b2e)

Reply:
At the time, I didn't think it was a likely scenario that could be possible unless the model completely failed or something, but now that you bring it up, I think it should at least be displayed. I'll go ahead and add it.